docs/evm.md

```diff
@@ -0,0 +1,18 @@
+
+INTRO
+Ethereum Virtual Machine (EVM) lies at the heart of every Ethereum activity.
+You can see it as the Operating System (OS) of Ethereum. But, as you don’t need to know how an OS works to use your laptop, you don’t need to know how EVM works to use Ethereum. That is surely why you’re reading this. 
+
+Kakarot is an implementation of the EVM in Starknet.
+
+The TL;DR
+The reason Ethereum Protocol exists is to maintain the Ethereum Virtual Machine. It is hard to imagine the future of Starknet without its own implementation of EVM, this is why we are building it.
```

Comment:
`The reason Ethereum Protocol exists is to maintain the Ethereum Virtual Machine.`

I'm not sure this is correct.

Reply:
> The Ethereum protocol itself exists solely for the purpose of keeping the continuous, uninterrupted, and immutable operation of this special state machine.

source: https://ethereum.org/en/developers/docs/evm/